crates/viewer/re_selection_panel/src/item_heading_with_breadcrumbs.rs

```diff
@@ -0,0 +1,317 @@
+//! The heading of each item in the selection panel.
+//!
+//! It consists of a blue background (selected background color),
+//! and within it there are "bread-crumbs" that show the hierarchy of the item.
+//!
+//! A > B > C > D > item
+//!
+//! Each bread-crumb is just an icon or a letter.
+//! The item is an icon and a name.
+//! Each bread-crumb is clickable, as is the last item.
+//!
+//! The bread crumbs hierarchy should be identical to the hierarchy in the
+//! either the blueprint tree panel, or the streams/time panel.
+
+use re_chunk::EntityPath;
+use re_data_ui::item_ui::{cursor_interact_with_selectable, guess_instance_path_icon};
+use re_entity_db::InstancePath;
+use re_log_types::EntityPathPart;
+use re_ui::{icons, list_item, DesignTokens, SyntaxHighlighting, UiExt as _};
+use re_viewer_context::{Contents, Item, SpaceViewId, ViewerContext};
+use re_viewport_blueprint::ViewportBlueprint;
+
+use crate::ItemTitle;
```

Comment:
see my other comment

```suggestion
use crate::item_title::ItemTitle
```